### ISSUE TITLE
AdminController::taskSaveas() does not trigger "onAdminSave"-Event

### ISSUE THREAD

**Issue by nmaier95** [getgrav/grav-plugin-admin#1544]
Hi there,

Is it on purpose the controller-action mentioned in the title doesn´t trigger the onAdminSave-Event as well as the normal save action?

Thanks in advance!

**Comment by rhukster** [comment]
Not that I can think of.  If you add this before the `$aPage->save()` in the `taskSaveas()` method:

```
$this->grav->fireEvent('onAdminSave', new Event(['page' => $obj]));
```

Do things work as expected?  I've not had chance to test.

**Comment by nmaier95** [comment]
I actually added the line by myself already to test it, and it did the job.

Furthermore I added an if-condition to check in both saving actions of the admincontroller if any hook called stopPropagation() on the event-object to not continue saving the page, for example if the user does not have the rights to save the currently selected locale. Don't know if this is wanted by the core-team of this plugin, but I'd do an pullrequest if so? Just let me know.

Else, the single line posted above is all I would suggest to add.

Greetings!
